.travis.yml

```diff
@@ -20,17 +20,14 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.6.*
```

Comment:
Will do. The travis config is also asking for php 5.4 and 5.5, although they're not being tested. 5.3 is tested however. Anything you'd like to do about this?